js/src/ui/Certifications/certifications.js

```diff
@@ -0,0 +1,83 @@
+// Copyright 2015, 2016 Ethcore (UK) Ltd.
+// This file is part of Parity.
+
+// Parity is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Parity is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Parity.  If not, see <http://www.gnu.org/licenses/>.
+
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+
+import { hashToImageUrl } from '../../redux/providers/imagesReducer';
+import { fetchCertifications } from '../../redux/providers/certifications/actions';
+
+import defaultIcon from '../../../assets/images/certifications/unknown.svg';
+
+import styles from './certifications.css';
+
+class Certifications extends Component {
+  static propTypes = {
+    account: PropTypes.string.isRequired,
+    certifications: PropTypes.object.isRequired,
+    dappsUrl: PropTypes.string.isRequired,
+
+    fetchCertifications: PropTypes.func.isRequired
+  }
+
+  componentWillMount () {
+    const { account, fetchCertifications } = this.props;
+    fetchCertifications(account);
+  }
+
+  render () {
+    const { account, certifications } = this.props;
+    const certificationsOfAccount = certifications[account] || [];
+
+    if (certificationsOfAccount.length === 0) {
+      return null;
+    }
+
+    return (
+      <div className={ styles.certifications }>
+        { certificationsOfAccount.map(this.renderCertification) }
+      </div>
+    );
+  }
+
+  renderCertification = (certification) => {
+    const { name, title, icon } = certification;
+    const { dappsUrl } = this.props;
+
+    const classNames = `${styles.certification} ${!icon ? styles.noIcon : ''}`;
+    const img = icon ? dappsUrl + hashToImageUrl(icon) : defaultIcon;
+    return (
+      <div className={ classNames } key={ name }>
+        <img className={ styles.icon } src={ img } />
+        <div className={ styles.text }>{ title || name }</div>
+      </div>
+    );
+  }
+}
+
+function mapStateToProps (state) {
+  return { certifications: state.certifications };
```

Comment:
Quick note about Redux state : for containers like this, we should use this as `mapStateToProps`:

```javascript
function mapStateToProps (_, initProps) {
  const { account } = this.props;

  return (state) => {
    const certification = state.certifications[account];
    return { certification };
  };
}
```

This is better because only the right `certification` Object is passed to the props, which means that the Component will not re-render when other certifications changes (which is the case if you pass the whole `state.certifications` Object).